src/web/app/components/question-types/question-statistics/question-statistics-calculation/constsum-recipients-question-statistics-calculation.ts

```diff
@@ -54,7 +62,13 @@ export class ConstsumRecipientsQuestionStatisticsCalculation
       const sum: number = answers.reduce((a: number, b: number) => a + b, 0);
       this.totalPointsPerOption[recipient] = sum;
       this.averagePointsPerOption[recipient] = +(answers.length === 0 ? 0 : sum / answers.length).toFixed(2);
+      this.averagePointsExcludingSelf[recipient] = this.averagePointsPerOption[recipient];
+      if (this.averagePointsPerOption[recipient] && pointsPerOptionToSelf[recipient] !== undefined) {
+        this.averagePointsExcludingSelf[recipient] =
```

Comment:
You do not need the ternary here since you already checked for `this.averagePointsPerOption[recipient]` not being zero

Reply:
The ternary expression checks for edge cases where the recipient has only one response which is also a self response!

If the ternary expression is removed, situations such as below can appear:
![image](https://user-images.githubusercontent.com/65604398/125190935-5b3ff780-e272-11eb-8a95-b31d691340ea.png)
because the only response received by Alice is a self-response

While I suppose such situations would be rare, I think it might be safer to leave the ternary expression there?
Or would it be better to shift the ternary expression outside of the if block?

Reply:
Ah good catch! Let's leave it then :)